sketch_expr/agm_sketch_expr.cpp

```diff
@@ -42,12 +42,17 @@ int main(int argc, char** argv) {
   auto end = std::chrono::steady_clock::now();
 
   std::chrono::duration<double> ins_time = end - start;
-  std::cout << "Time to update a sketch of length " << n 
-            << " with " << updates << " updates was: " 
-            << ins_time.count() << std::endl;
 
-  std::cout << "Updates per second: " << updates / ins_time.count() << std::endl;
-  std::cout << "Sketch is " << sketch.size() << " bytes\n";
+  // for posterity
```

Comment:
Why did we reformat the output for this test?

Reply:
Just length? We didn't reformat the other tests

Reply:
it’s very jank. the experiment cats stdout to a file which we used to have to manually comb through to get results. we can’t have it output to a final csv since we need to do post processing on the combined runs of agm and cube. this is the least bad way I saw of doing it: sacrifice some readability to streamline csv creation

Reply:
Gotcha, so we're parsing these stdout results. That makes sense.